midterm/Neumannova_midterm.R

```diff
@@ -0,0 +1,360 @@
+
+share <- (males_basketball_2012 / males_athletes_2012) * 100
+rounded_share <- round(share[[1]], 1)
+
+cat("The share of male basketball players among all male athletes in 2012 was", rounded_share[[1]], "%")
+
+# 3.3)
+# What are the mean and standard deviation of height for female tennis players who participated in the 2000
+# Olympics. Round to the first decimal
+
+athletes %>%
+  filter(Sex == "F", Year == 2000, Sport == "Tennis") %>%
+  summarise(mean_height = round(mean(Height, na.rm = T), 1), standard_deviation_height = round(sd(Height, na.rm = T), 1))
+
+
+# 3.4)
+# Find the heaviest athete among 2006 Olympics participants. What sport did he or she do?
+
+heavy_sport <- athletes %>%
+  filter(Year == 2006) %>%
+  filter(Weight == max(Weight, na.rm = T)) %>%
+  pull(Sport)
+
+cat("The heaviest athlete in 2016 did", as.character(heavy_sport))
+
+# 3.5)
+# How many time did John Aalberg participate in the Olympics held in different years
+
+JA_years <- athletes %>%
+  group_by(Year) %>%
+  filter(Name == "John Aalberg") %>%
+  summarise(number_of_different_years = length(unique(Year))) %>%
+  nrow()
+
+cat("John Aalberg participated in the Olympics in", JA_years, "different years")
+
+# 3.6)
+# How many gold medals in tennis did the Switzerland team win at the 2008 Olympics?
+
+Swiss_gold_2008 <- athletes %>%
+  filter(Year == 2008, Team == "Switzerland", Medal == "Gold", Sport == "Tennis") %>%
+  nrow()
+
+cat("In 2008, Switzerland won", Swiss_gold_2008, "gold medals in tennis")
+
+# 3.7)
+# Is it true that there were Summer Olympics held in Atlanta? Is it true that there were Winter Olympics
+# held in Squaw Valley
+
+Atlanta_summer <- athletes %>%
+  filter(City == "Atlanta", Season == "Summer") %>%
+  nrow()
+
+Determine_Atlanta <- Atlanta_summer > 0
+cat("There were Summer Games in Atlanta.", Determine_Atlanta)
+
+Squaw_Valley_winter <- athletes %>%
+  filter(City == "Squaw Valley", Season == "Winter") %>%
+  nrow()
+
+Determine_Squaw_Valley <- Squaw_Valley_winter > 0
+cat("There were Winter Games in Squaw Valley.", Determine_Squaw_Valley)
+
+# 3.8)
+# Is it true that Spain won fewer medals than Italy at the 2016 Olymptics?
+
+Spain <- athletes %>%
+  na.omit() %>%
+  filter(Year == 2016, Team == "Spain") %>%
+  nrow()
+
+Italy <- athletes %>%
+  na.omit() %>%
+  filter(Year == 2016, Team == "Italy") %>%
+  nrow()
+
+Determine_SP_IT <- Spain < Italy
+cat("Spain won fewer medals than Italy in 2016.", Determine_SP_IT)
+
+# 3.9)
+# What is the absolute difference between the number of unique sports at the 1986 Olympics and 2002 Olympics?
+
+# There were no Games in 1986, therefore, the following function will give 0
+
+Sports_1986 <- athletes %>%
+  filter(Year == 1986) %>%
+  nrow()
+
+Sports_2002 <- athletes %>%
+  filter(Year == 2002) %>%
+  pull(Sport) %>%
+  unique() %>%
+  length()
+
+difference = abs(Sports_1986 - Sports_2002)
+cat("The absolute difference between the number of unique sports at 1986 and 2002 is", difference)
```

Comment:
Overall very good impression.

You've made a decent progress during the course. 

Your score is 100/100. Congratz